src/modules/flight_mode_manager/tasks/Auto/FlightTaskAuto.cpp

```diff
@@ -827,8 +827,13 @@ void FlightTaskAuto::_updateTrajConstraints()
 
 	} else { // down
 		_position_smoothing.setMaxAccelerationZ(_param_mpc_acc_down_max.get());
-		_position_smoothing.setMaxVelocityZ(_param_mpc_z_vel_max_dn.get());
+		_position_smoothing.setMaxVelocityZ(_param_mpc_z_v_auto_dn.get());
 	}
+
+	// Stretch the constraints of the velocity controller to leave some room for an additional
+	// correction required by the altitude controller
```

Comment:
```suggestion
	// correction required by the altitude/vertical position controller
```
Maybe more clear, not sure